R/class-plotdata.R

```diff
@@ -86,6 +101,7 @@ newPlotdata <- function(.dt = data.table(),
     veupathUtils::logWithTime(paste('Replaced NA with 0 in the following columns: ', impute0cols), verbose)
   }
 
+  #lat and lon must be used w a geohash, so they dont need to be part of completeCases*
```

Comment:
i dont follow this part - why does being used with a geohash make a difference here?

Reply:
my understanding from bob is that if there is a geohash, there is latitude and longitude.

Reply:
tbh this assumption in combination w the fact that any time we use geohash we seem to also need lat and lon make me think the three might be a candidate for a class when we switch to s4.